### PR TITLE
Plenty 182 rest exporter fetch only domain name

### PR DESCRIPTION
**Pull request by darius-adeoweb**

### CODE REVIEW

File: src/Helper/Url.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+namespace Findologic\Plentymarkets\Helper;
+
+class Url
+{
+    public static function getHost(string $url): string
+    {
+        $parseUrl = parse_url(trim($url));
+
+        if (!empty($parseUrl['host'])) {
```
Comment (TheKeymaster) [inline src/Helper/Url.php]: When is the `host` empty and when do we need to manually parse the host by dropping everything after `/`?

File: tests/Parser/ParserAbstractTest.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+use Findologic\Plentymarkets\Parser\ParserAbstract;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+class ParserAbstractTest extends TestCase
+{
+    public function correctDomainUrlProvider(): array
+    {
+        return [
+            ['www.example.com/lt', 'www.example.com'],
```
Comment (TheKeymaster) [inline tests/Parser/ParserAbstractTest.php]: What happens if we put in an URL that is no URL? Can we throw an exception and ensure that it is thrown if that's the case?

File: src/Helper/Url.php (hunk 1 above)
Comment (darius-adeoweb) [reply src/Helper/Url.php]: If `$url` does not have protocol (`www.example.com`), `parse_url` returns an associative array that does not have `host` value. If protocol is set, then `host` value is set.

File: tests/Parser/ParserAbstractTest.php [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+use Findologic\Plentymarkets\Parser\ParserAbstract;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+class ParserAbstractTest extends TestCase
+{
+    public function correctDomainUrlProvider(): array
+    {
+        return [
+            ['http:///example.com', false],
```
Comment (TheKeymaster) [inline tests/Parser/ParserAbstractTest.php]: Split this up into two tests (two data providers). A test should only test the same output. In this case we are ensuring an exception is thrown, which is different then asserting that the domain was parsed correctly.

**Pull request merged**